File: src/java/org/voegtle/weatherstation/server/data/UnformattedWeatherDTO.java
package org.voegtle.weatherstation.server.data;

import java.io.Serializable;
import java.util.Date;

public class UnformattedWeatherDTO implements Serializable {
  private Date time;
  private Float temperature;
  private Float humidity;
  boolean raining;
  private Float rainLastHour;
  private Float rainToday;
  private Float windspeed;

  public UnformattedWeatherDTO() {
  }

  public Date getTime() {
    return time;
  }

  public void setTime(Date time) {
    this.time = time;
  }

  public Float getTemperature() {
    return temperature;
  }

  public void setTemperature(Float temperature) {
    this.temperature = temperature;
  }

  public Float getHumidity() {
    return humidity;
  }

  public void setHumidity(Float humidity) {
    this.humidity = humidity;
  }

  public boolean isRaining() {
    return raining;
  }

  public void setRaining(boolean raining) {
    this.raining = raining;
  }

  public Float getRainLastHour() {
    return rainLastHour;
  }

  public void setRainLastHour(Float rainLastHour) {
    this.rainLastHour = rainLastHour;
  }

  public Float getWindspeed() {
    return windspeed;
  }

  public void setWindspeed(Float windspeed) {
    this.windspeed = windspeed;
  }

  public Float getRainToday() {
    return rainToday;
  }

  public void setRainToday(Float rainToday) {
    this.rainToday = rainToday;
  }

}

File: src/java/org/voegtle/weatherstation/server/data/UnformattedWeatherDTO.kt
package org.voegtle.weatherstation.server.data

import java.util.*

data class UnformattedWeatherDTO(
    val time: Date,
    val localTime: String,
    val temperature: Float,
    val insideTemperature: Float? = null,
    val humidity: Float,
    val isRaining: Boolean? = null,
    val rainLastHour: Float? = null,
    val rainToday: Float? = null,
    val windspeed: Float? = null,
    val insideHumidity: Float? = null,
    val watt: Float? = null)
